mysql-test/suite/encryption/r/bug_ps_6922.result

```diff
@@ -0,0 +1,8 @@
+# Below queries would crash the server. Check that they work OK now.
```

Comment:
Please add this to `mysql-test/t/percona_expand_fast_index_creation.test` instead.

Reply:
Good suggestion to reuse existing feature based test file :)

Reply:
done